etna/pipeline/mixins.py

```diff
@@ -0,0 +1,82 @@
+from copy import deepcopy
+from typing import Sequence
+
+import numpy as np
+import pandas as pd
+from typing_extensions import get_args
+
+from etna.datasets import TSDataset
+from etna.models import ModelType
+from etna.models import NonPredictionIntervalContextIgnorantAbstractModel
+from etna.models import NonPredictionIntervalContextRequiredAbstractModel
+from etna.models import NonPredictionIntervalModelType
+from etna.models import PredictionIntervalContextIgnorantAbstractModel
+from etna.models import PredictionIntervalContextRequiredAbstractModel
+from etna.transforms import Transform
+
+
+class ModelPipelinePredictMixin:
```

Comment:
May be just `PipelinePredictMixin`? What do we need prefix "Model" for?

Reply:
Because it only for pipelines that has models inside it. Ensembles are pipelines too, but they don't have model inside it.

Reply:
Got it